src/nav/style-guide.yml

```diff
@@ -1,119 +1,128 @@
-      - title: Delete a document
-        path: /docs/style-guide/processes-procedures/delete-document
-      - title: Embed images
-        path: /docs/style-guide/processes-procedures/embed-images
-      - title: Embed videos
-        path: /docs/style-guide/processes-procedures/embed-videos
-      - title: Update the home page
-        path: /docs/style-guide/processes-procedures/edit-homepage
-  - title: Tech writer processes
-    path: /docs/style-guide/writer-workflow
-    pages:
-      - title: Get around GitHub
-        path: /docs/style-guide/writer-workflow/github-intro
-      - title: Tech writer workflow
-        path: /docs/style-guide/writer-workflow/tech-writer-workflow
-      - title: Peer editor workflow
-        path: /docs/style-guide/writer-workflow/peer-editor-workflow
-      - title: GitHub troubleshooting
-        path: /docs/style-guide/writer-workflow/github-troubleshooting
-  - title: Article templates
-    path: /docs/style-guide/article-templates
-    pages:
-      - title: Basic doc template
-        path: /docs/style-guide/article-templates/basic-doc-template
-      - title: Agent release notes template
-        path: /docs/style-guide/article-templates/agent-release-notes-template-123
-      - title: What's new? guidelines
-        path: /docs/style-guide/article-templates/whats-new-style-guidelines
-      - title: Troubleshooting docs guide
-        path: /docs/style-guide/article-templates/troubleshooting-docs-guide
-      - title: Data dictionary style guidelines
-        path: /docs/style-guide/article-templates/data-dictionary-style-guidelines
-      - title: API tutorial template
-        path: /docs/style-guide/article-templates/api-tutorial-template
-      - title: Agent API guide template
-        path: /docs/style-guide/article-templates/agent-api-guide-template
-      - title: apiStyleGuidelines (Example agent API)
-        path: /docs/style-guide/article-templates/apistyleguidelines-example-agent-api
-      - title: GraphQL API tutorial template
-        path: /docs/style-guide/article-templates/graphql-api-tutorial-template
-      - title: Landing page template
-        path: /docs/style-guide/article-templates/landing-page-template
-      - title: Agent release notes template 1.2.3
-        path: /docs/style-guide/article-templates/agent-release-notes-template-123
+    - title: Writing great content at New Relic
+      path: /docs/style-guide/writing-strategies
+      pages: 
+        - title: Introduction to the style guide
+          path: /docs/style-guide/writing-strategies/introduction-style-guide
+        - title: Voice, tone, and style
+          path: /docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic
+    - title: Capitalization
+      path: /docs/style-guide/capitalization
+      pages:
+        - title: Capitalization
+          path: /docs/style-guide/capitalization/capitalization
+    - title: Formatting
```

Comment:
There's a lot of content in "Formatting". Maybe it's worth considering breaking it up into slightly more granular categories?

Just a loose, back-of-the-napkin suggestion:

+ Text formatting (could include hyperlinks and headings)
+ Code formatting
+ UI formatting
+ Lists and tables (could include collapsers)

Reply:
I like this idea! Maybe a separate category about structure, which could address things like headings and subheadings, length of page, use of lists, tables, collapsers

Reply:
@barbnewrelic Structure-related docs is great! I was struggling to come up with a better category. It makes a lot of sense to divide things between structure and formatting.